src/neptune/metadata_containers/metadata_container.py

```diff
@@ -380,6 +380,20 @@ def stop(self, *, seconds: Optional[Union[float, int]] = None) -> None:
             self._state = ContainerState.STOPPED
             self._forking_cond.notify_all()
 
+    def get_state(self) -> str:
+        """Returns the current state of the container as a string.
+
+        Examples:
+            >>> from neptune import init_run
+            >>> run = init_run()
+            >>> run.get_state()
+            'started'
+            >>> run.stop()
+            >>> run.get_state()
+            'stopped'
```

Comment:
For consistency, we should change the returned values to `active` and `inactive` if possible